README.md

```
@@ -126,6 +126,156 @@ podman stop ssvirt-ui-container
+- configMapRef:
+    name: ssvirt-ui-config
+```
+
+Edit the ConfigMap to match your environment:
+
+```bash
+kubectl edit configmap ssvirt-ui-config
+```
+
+Available configuration options:
+- `API_BASE_URL`: Backend API endpoint URL (default: "http://ssvirt-backend:8080/api")
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)